inst/tests/tests.Rraw

```diff
@@ -18004,7 +18004,7 @@ test(2233.34, copy(DT)[, same_value:=value[1],    by=.(by1, by2), verbose=TRUE],
 test(2233.35, copy(DT)[, same_value:=value[1],    by=.(by2, by1), verbose=TRUE], ans,                                output=out)
 test(2233.36, copy(DT)[, same_value:=value[1], keyby=.(by2, by1), verbose=TRUE], setkey(ans,by2,by1),                output=out)
 # similar to #5307 using integer
-DT = data.table(A=INT(2,1,2,1), B=6:3, v=11:14)                                                
+DT = data.table(A=INT(2,1,2,1), B=6:3, v=11:14)
```

Comment:
Changes like that should go into separate PR, that helps for merge conflicts resolution. Just letting you know, don't change it.

Reply:
I'm indifferent -- this is the boy scout principle

https://deviq.com/principles/boy-scout-rule

it seems unlikely to cause merge conflicts, unless the next PR also fails to fix the whitespace issue

Reply:
Apologies. Inadvertent consequence of using a keyboard shortcut to run tests.

Reply:
What works for me very good is to verify git diff --cached just before commiting